kie-server-parent/kie-server-tests/kie-server-integ-tests-controller/src/test/java/org/kie/server/integrationtests/controller/KieControllerManagementIntegrationTest.java

```diff
@@ -418,4 +557,27 @@ protected ServerTemplate createServerTemplate() {
 
         return serverTemplate;
     }
+
+    protected void checkContainerConfig(String serverTemplateId, String containerId, ContainerConfig... configs) {
+        Map<Capability, ContainerConfig> configMap = controllerClient.getContainerInfo(serverTemplateId, containerId).getConfigs();
```

Comment:
JSON marshalling issue. When try check configuration, in unmarshal it to only the main type without actual type.